<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="menuLayout">
      <item>
       <widget class="QPushButton" name="pbMenu">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: grey;
    border-style: outset;
    border-width: 1px;
    border-radius: 9px;
    border-color: black;
    padding: 1px;
}
QPushButton:checked {
	background-color: rgb(58, 58, 58);
}
</string>
        </property>
        <property name="text">
         <string>MENU</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbShift">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: grey;
    border-style: outset;
    border-width: 1px;
    border-radius: 9px;
    border-color: black;
    padding: 1px;
}
QPushButton:checked {
	background-color: rgb(58, 58, 58);
}
</string>
        </property>
        <property name="text">
         <string>Shift</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbSelect">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: grey;
    border-style: outset;
    border-width: 1px;
    border-radius: 9px;
    border-color: black;
    padding: 1px;
}
QPushButton:checked {
	background-color: rgb(58, 58, 58);
}
</string>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbStart">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: grey;
    border-style: outset;
    border-width: 1px;
    border-radius: 9px;
    border-color: black;
    padding: 1px;
}
QPushButton:checked {
	background-color: rgb(58, 58, 58);
}
</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="padLayout">
      <item row="2" column="6">
       <widget class="QPushButton" name="pbB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: grey;
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: black;
    min-width: 1.5em;
    padding:4 px;
}
QPushButton:checked {
	background-color: rgb(58, 58, 58);
}</string>
        </property>
        <property name="text">
         <string>B</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pbRight">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: grey;
	border-color: black;
   border-style: outset;
   border-width: 1px;
}
QPushButton:checked {
	background-color: rgb(58, 58, 58);
}
</string>
        </property>
        <property name="text">
         <string>→</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pbDown">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: grey;
	border-color: black;
   border-style: outset;
   border-width: 1px;
}
QPushButton:checked {
	background-color: rgb(58, 58, 58);
}
</string>
        </property>
        <property name="text">
         <string>↓</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="pbA">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: grey;
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: black;
    min-width: 1.5em;
    padding:4 px;
}
QPushButton:checked {
	background-color: rgb(58, 58, 58);
}</string>
        </property>
        <property name="text">
         <string>A</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pbUp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: grey;
	border-color: black;
   border-style: outset;
   border-width: 1px;
}
QPushButton:checked {
	background-color: rgb(58, 58, 58);
}
</string>
        </property>
        <property name="text">
         <string>↑</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="pbY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: grey;
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: black;
    min-width: 1.5em;
    padding:4 px;
}
QPushButton:checked {
	background-color: rgb(58, 58, 58);
}</string>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pbLeft">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: grey;
	border-color: black;
   border-style: outset;
   border-width: 1px;
}
QPushButton:checked {
	background-color: rgb(58, 58, 58);
}
</string>
        </property>
        <property name="text">
         <string>←</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="pbX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: grey;
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: black;
    min-width: 1.5em;
    padding:4 px;
}
QPushButton:checked {
	background-color: rgb(58, 58, 58);
}</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
